bugzilla/commentsdisk.go

```diff
@@ -160,6 +160,24 @@ func (s *CommentDiskStore) Sync(keys []string) ([]*BugComments, error) {
 	return bugs, nil
 }
 
+func (s *CommentDiskStore) DeleteBug(bug *Bug) error {
+	_, path := s.pathForBug(bug)
+	return os.Remove(path)
+}
+
+func (s *CommentDiskStore) CloseBug(bug *Bug) error {
+
+	_, path := s.pathForBug(bug)
+	bugComments, err := readBugComments(path)
```

Comment:
Why is this?  Why aren't you getting it from the store for comments?

Reply:
because it could be removed from the store already.  I suppose I can make the attempt before reading from disk, but i wouldn't want to assume it's guaranteed to be there.